docs/customer/auth.md

```diff
@@ -92,12 +92,34 @@ ___
 
 ## SIP User Authentication
 
-When you enable **SIP Authentication**, ConnexCS will reject the initial SIP INVITE with a "407 Authentication Required". This message includes a 'nonce' (a uniquely randomly generated number, which is hashed). The customer switch will send appropriate authentication information to ConnexCS, which will connect the call.
+When you enable **SIP Authentication**, ConnexCS will reject the initial SIP INVITE with a "407 Authentication Required". This message includes a 'nonce' (a uniquely randomly generated hashed number). The customer switch will send appropriate authentication information to ConnexCS, which will connect the call.
 
 Generic SIP Trace showing the Challenge Response:
 
 &emsp; ![alt text][407-trace]
 
+```mermaid
+    sequenceDiagram
+    autonumber
+    Alice->>Bob: INVITE
+    Bob-->>Alice: 100 Trying
+    Bob->>Alice: 407 Proxy Authentication Required
+    Note over Alice,Bob: 407 contains nonce.
+    Note left of Alice: HASH(Combine Password + Nonce).
+    Alice->>Bob: INVITE (+ Auth Header)
+    Bob-->>Alice: 100 Trying
+    Note right of Bob: HASH(Combine Password + Nonce).
+    Note right of Bob: Compare Hashes - They Match.
+    Bob-->>Alice: 183 Ringing
+    Bob->>Alice: 200 OK (Connected)
+```
+
+For call authentication we should have a Username and a Password. The Username and Password should get to the other side.
+
+The Username is sent on Plain-text and the user (Alice) hashes the password. [**407 Proxy Authentication**](https://en.wikipedia.org/wiki/List_of_SIP_response_codes#4xx%E2%80%94Client_Failure_Responses) contains a nonce. A nonce is a random String of which gets send over to Alice. Both Alice and Bob are aware of this random string. Authorization header is sent with the INVITE. Then Bob combines the password with the nonce and compares the nonce. If the hashes match, the call gets connected.
+
+!!! note "407 Proxy Authentication is a part of Challenge-Response and is necessary when you proceed with SIP User Auth. Also you cannot have IP Authentication and SIP Authentication work together"
```

Comment:
Please change to "note "407 Proxy Authentication is a part of Challenge-Response and is necessary when you proceed with SIP User Auth. Also you cannot have IP Authentication and **User/Password** Authentication work together"